ckanext/security/tests/test_validators.py

```diff
@@ -0,0 +1,112 @@
+import ckan.model as model
+import ckan.tests.factories as factories
+import pytest
+from ckan.lib.navl.dictization_functions import Invalid
+from ckan.plugins import toolkit as tk
+from passlib.handlers.pbkdf2 import pbkdf2_sha512
+
+import ckanext.security.validators as validators
+from ckanext.security.constants import PLUGIN_EXTRAS_TABULIST_KEY
+
+
+class TestValidators(object):
+
+    def test_password_length_default(self):
+        """Check if the default password length is set to 10"""
+        # WHEN
+        actual = validators._min_password_length()
+        # THEN
+        assert actual == 10
+
+    @pytest.mark.ckan_config(u'ckanext.security.min_password_length', u'12')
+    def test_password_length_config(self):
+        """Check if the password length is set to 12 when set via config"""
+        # WHEN
+        actual = validators._min_password_length()
+        # THEN
+        assert actual == 12
+
+    @pytest.mark.ckan_config(u'ckanext.security.min_password_length', u'12')
+    def test_password_length_is_validated_ok(self):
+        """Check if the password length is validated correctly"""
+        # WHEN + THEN
+        validators.user_password_validator("pw", {"pw": "Aa_123456789"}, None, None)
+
+    @pytest.mark.ckan_config(u'ckanext.security.min_password_length', u'12')
+    def test_invalid_password_length_is_too_short(self):
+        """Check if the password length is validated correctly"""
+        # WHEN + THEN
+        with pytest.raises(Invalid):
+            validators.user_password_validator("pw", {"pw": "Aa_123"}, None, None)
```

Comment:
This should test a range of cases, especially edge cases. Eg if the minimum is 12, we should verify that a 12-character password is okay and 11 is not. Consider using a parameterised test.